fluent-mechanical/wf_fm_02_mechanical.py

```diff
@@ -0,0 +1,947 @@
+    "File1:Heat Transfer Coefficient@F",
+    "File2:Heat Transfer Coefficient@F",
+    "File3:Heat Transfer Coefficient@F",
+]
+Amb_Temp = ["File1:Temperature@E","File2:Temperature@E","File3:Temperature@E"]
+Ana_time = [
+    "0",
+    "1e-3",
+    "2e-3",
+    "20",
+    "30",
+    "320",
+    "330",
+    "350",
+    "360",
+    "380",
+    "390",
+    "680",
+    "690",
+    "710",
+    "720",
+]
+
+for i in range(numofsteps-1):
+    table.Add(None)
+
+table[0][0] = Film_Coeff[2]
+table[0][1] = Amb_Temp[2]
+table[0][2] = Ana_time[0]
+
+table[1][0] = Film_Coeff[2]
+table[1][1] = Amb_Temp[2]
+table[1][2] = Ana_time[1]
+
+table[2][0] = Film_Coeff[2]
+table[2][1] = Amb_Temp[2]
+table[2][2] = Ana_time[2]
+
+table[3][0] = Film_Coeff[2]
+table[3][1] = Amb_Temp[2]
+table[3][2] = Ana_time[3]
+
+table[4][0] = Film_Coeff[0]
+table[4][1] = Amb_Temp[0]
+table[4][2] = Ana_time[4]
+
+table[5][0] = Film_Coeff[0]
+table[5][1] = Amb_Temp[0]
+table[5][2] = Ana_time[5]
+
+table[6][0] = Film_Coeff[1]
+table[6][1] = Amb_Temp[1]
+table[6][2] = Ana_time[6]
+
+table[7][0] = Film_Coeff[1]
+table[7][1] = Amb_Temp[1]
+table[7][2] = Ana_time[7]
+
+table[8][0] = Film_Coeff[2]
+table[8][1] = Amb_Temp[2]
+table[8][2] = Ana_time[8]
+
+table[9][0] = Film_Coeff[2]
+table[9][1] = Amb_Temp[2]
+table[9][2] = Ana_time[9]
+
+table[10][0] = Film_Coeff[0]
+table[10][1] = Amb_Temp[0]
+table[10][2] = Ana_time[10]
+
+table[11][0] = Film_Coeff[0]
+table[11][1] = Amb_Temp[0]
+table[11][2] = Ana_time[11]
+
+table[12][0] = Film_Coeff[1]
+table[12][1] = Amb_Temp[1]
+table[12][2] = Ana_time[12]
+
+table[13][0] = Film_Coeff[1]
+table[13][1] = Amb_Temp[1]
+table[13][2] = Ana_time[13]
+
+table[14][0] = Film_Coeff[2]
+table[14][1] = Amb_Temp[2]
+table[14][2] = Ana_time[14]
+
+selection = NS_GRP.Children[4]
+imported_convection_62.Location = selection
+imported_load_id = imported_convection_62.ObjectId
+imported_load = DataModel.GetObjectById(imported_load_id)
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# imported_convection_62.InternalObject.ExternalLoadSteppingType = 1
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+imported_load.ImportLoad()
+
+Tree.Activate([imported_load])
+ExtAPI.Graphics.Camera.SetFit()
+ExtAPI.Graphics.ExportImage(
+    os.path.join(project_directory, "imported_temperature.png"), image_export_format, settings_720p
+)
+"""
+)
+mechanical.download(
+    files=os.path.join(project_directory, "imported_temperature.png"), target_dir=OUTPUT_DIR
+)
+if GRAPHICS_BOOL:
+    display_image("imported_temperature.png")
+
+###############################################################################
+# Solve and post-process the results
+# ----------------------------------
+#
+mechanical.run_python_script(
+    """
+# Insert results objects
+
+Temp = TRANS_THERM_SOLN.AddTemperature()
+Temp.DisplayTime = Quantity("680 [s]")
+
+# Run Solution: Transient Thermal Simulation
+
+#TRANS_THERM_SOLN.Solve(True)
```

Comment:
```suggestion
TRANS_THERM_SOLN.Solve(True)
```

Will uncomment this once entire workflow script is ready